test/TextForEvent-test.ts

```diff
@@ -444,4 +446,44 @@ describe('TextForEvent', () => {
             expect(textForEvent(messageEvent)).toEqual('@a: test message');
         });
     });
+
+    describe("textForCallEvent()", () => {
+        let mockClient: MatrixClient;
+        let callEvent: MatrixEvent;
+
+        beforeEach(() => {
+            mockClient = createTestClient();
+
+            MatrixClientPeg.get = () => mockClient;
```

Comment:
This is the same thing as `stubClient(); mockClient = MatrixClientPeg.get()`, ftr